NanoProd/customiseNano.py

```diff
@@ -40,11 +40,238 @@ def customise(process):
+      process.fatJetTable.variables.PNET_Htm = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probHtm')",float,precision=10)
+      process.fatJetTable.variables.PNET_Hte = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probHte')",float,precision=10)
+      process.fatJetTable.variables.PNET_Hbb = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probHbb')",float,precision=10)
+      process.fatJetTable.variables.PNET_Hcc = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probHcc')",float,precision=10)
+      process.fatJetTable.variables.PNET_Hqq = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probHqq')",float,precision=10)
+      process.fatJetTable.variables.PNET_Hgg = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probHgg')",float,precision=10)
+      process.fatJetTable.variables.PNET_QCD2hf = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probQCD2hf')",float,precision=10)
+      process.fatJetTable.variables.PNET_QCD1hf = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probQCD1hf')",float,precision=10)
+      process.fatJetTable.variables.PNET_QCD0hf = Var("bDiscriminator('pfParticleNetAK8LastJetTags:probQCD0hf')",float,precision=10)
+      process.fatJetTable.variables.PNET_masscorr = Var("bDiscriminator('pfParticleNetAK8LastJetTags:masscorr')",float,precision=10)
+      process.fatJetTable.variables.PNET_massregression = Var("bDiscriminator('pfParticleNetMassRegressionJetTags:mass')",float,precision=10)
+
+      process.edTask.add(getattr(process,"slimmedJetsAK8UpdatedPNET"))
+      process.edTask.add(getattr(process,"pfParticleNetAK8JetTagInfos"))
+      process.edTask.add(getattr(process,"pfParticleNetMassRegressionJetTags"))
+      process.edTask.add(getattr(process,"pfParticleNetAK8LastJetTags"))
+      process.edTask.add(getattr(process,"pfParticleNetAK8LastJetTagInfos"))
+
+    # For some reason these are lost when integrated as customization (even when not processing AK8), so re-add them to path.
+    #process.edTask.add(getattr(process,"updatedPatJetsAK8WithDeepInfo"))
+    #process.edTask.add(getattr(process,"selectedUpdatedPatJetsAK8WithDeepInfo"))
+    #process.edTask.add(getattr(process,"updatedPatJetsTransientCorrectedAK8WithDeepInfo"))
+    #process.edTask.add(getattr(process,"patJetCorrFactorsTransientCorrectedAK8WithDeepInfo"))
+    #process.edTask.add(getattr(process,"pfParticleNetTagInfosAK8WithDeepInfo"))
+    #process.edTask.add(getattr(process,"pfParticleNetMassRegressionJetTagsAK8WithDeepInfo"))
+    #process.edTask.add(getattr(process,"patJetCorrFactorsAK8WithDeepInfo"))
+    for key in process.__dict__.keys():
+        if(type(getattr(process,key)).__name__=='EDProducer' or type(getattr(process,key)).__name__=='EDFilter') and "AK8WithDeepInfo" in key:
+            process.edTask.add(getattr(process,key))
+
+    process.edPath = cms.Path(process.edTask)
+
+    # Schedule definition
+    process.schedule = cms.Schedule(process.edPath,process.nanoAOD_step,process.endjob_step,process.NANOAODSIMoutput_step)
```

Comment:
would it be possible to insert `edPath` without redefining the whole schedule? Something like
```
process.schedule.insert(0, edPath)
```